src/lib.rs

```diff
@@ -388,9 +388,13 @@ pub fn get_entry(entry_hash: HashString) -> Result<Option<String>, RibosomeError
 }
 
 /// FIXME DOC
-pub fn link_entries<S: Into<String>>(_base: HashString, _target: HashString, _tag: S) {
+pub fn link_entries<S: Into<String>>(
+    _base: HashString,
+    _target: HashString,
+    _tag: S
+) -> Result<(), RibosomeError> {
```

Comment:
Is this the right function signature? I was thinking so, because if it comes back as Ok, that's enough to know it worked, no return value needed

Reply:
I think so.
I mean, also link entries are committed and have an address.. that could be returned, but is actually not really needed. I would keep it like this for now..